android/build.gradle.kts

```diff
@@ -11,11 +11,29 @@ rootProject.layout.buildDirectory.value(newBuildDir)
 subprojects {
     val newSubprojectBuildDir: Directory = newBuildDir.dir(project.name)
     project.layout.buildDirectory.value(newSubprojectBuildDir)
+    
+    // Configure JVM target for all subprojects to ensure consistency
+    afterEvaluate {
+        if (project.hasProperty("android")) {
+            extensions.configure<com.android.build.gradle.BaseExtension> {
+                compileOptions {
```

Comment:
Remove this change and run flutter clean and try to run again. This may not be necessary

Reply:
We can remove for now. If the code runs without it, we don't need it